mltsp/parallel_processing.py

```diff
@@ -422,12 +453,26 @@ def featurize_in_parallel(headerfile_path, zipfile_path, features_to_use=[],
     zipfile = tarfile.open(zipfile_path)
     zipfile.extractall(tmp_dir_path)
     all_fnames = zipfile.getnames()
+    all_fnames = [f for f in all_fnames if not os.path.isdir(f)]
+    if is_test:
+        all_fnames = all_fnames[:3]
+
+    orig_fnames_dict = {}
+    tags = []
+    for i in range(len(all_fnames)):
+        short_fname = ntpath.basename(all_fnames[i])
+        tags.append(str(session_key +
+                        short_fname.replace(".", "_")))
+        orig_fnames_dict[short_fname.replace(".", "_")] = short_fname
+        if not os.path.isabs(all_fnames[i]):
+            all_fnames[i] = os.path.join(tmp_dir_path, all_fnames[i])
+    # Push all data files to DDFS
+    print(all_fnames)
```

Comment:
Debug statement, or meant to be here?  If so, perhaps annotate with some user-readable prefix.